impl/client/src/main/javascript/web/prompting/components/ScrollingPromptPanelLayoutComponent.js

```diff
@@ -15,26 +15,31 @@
  *
  */
 
-define(['./PromptLayoutComponent', 'common-ui/jquery-clean'], function(PromptLayoutComponent, $) {
+define(["./PromptLayoutComponent", "common-ui/jquery-clean"], function(PromptLayoutComponent, $) {
   return PromptLayoutComponent.extend({
-    update: function () {
-      if (this.components) {
-        var $htmlObject = $('#' + this.htmlObject);
+    update: function() {
+      if(this.components) {
```

Comment:
Missing space after this `if` statement and all others. Please check with linter

Reply:
I thought common-ui does not have any rules for linter

Reply:
hmm, you are right, that probably explains why each file follows its own conventions.. very confusing

Reply:
Yeah most of the files on Common-Ui don't have spaces after `if` statements. I had a lot a troubles at first with it because VSCode was applying those changes automatically when saving the files.